### PR TITLE
Add greeting from Roman and Semyon =)

### PR DESCRIPTION
**Pull request by Tsapfik**
Hi kottans!
Thanks for your work and such a great course. I hope I can get to Stage 1.

### CODE REVIEW

**Review by lenadgit — APPROVED** [review]
Nice job. You may begin to do the next task, it is not necessary to wait for approval of a current task

**Comment by Tsapfik** [comment]
> Nice job. You may begin to do the next task, it is not necessary to wait for approval of a current task

Thanks a lot for quik review of my PR :smiley_cat: Next task is in progress :rocket: 
P.S. Sorry for my English if you found some mistake, i'm still learning.

**Pull request merged**
